spaces/verified.json

```
@@ -101,6 +101,20 @@
   "evmaverick.eth": 1,
   "livelabs.eth": 1,
   "jadeprotocol.eth": 1,
+    "dea.eth": 1,
+  "pooltogether.eth": 1,
```

@metrox-eth Could you reindent this line? Thanks
```suggestion
  "jadeprotocol.eth": 1,
  "dea.eth": 1,
  "pooltogether.eth": 1,
```